guides/bidirectionality.md

```diff
@@ -0,0 +1,22 @@
+# Angular Material bi-directionality
+
+### How to set your application direction
+The simplest way is applying `dir` attribute on either the `html` or `body` tags.
```

Comment:
This describes the behavior right now. It could potentially change in a subsequent iteration.